client.go

```diff
@@ -307,6 +319,13 @@ func (client *NativeClient) Session() (*ssh.Session, *ssh.Client, *ssh.Client, e
 	return session, proxy, conn, nil
 }
 
+//RemoteOutput returns the of the command run proxied through the host the remote host. The same credentials and timeout are used.  The remote key must be in the home directory
```

Comment:
returns the of the command -> returns the output of the command

through the host the remote host -> through the host to the remote host?